libs/langchain-community/src/vectorstores/pgvector.ts

```diff
@@ -113,6 +119,12 @@ export class PGVectorStore extends VectorStore {
       : `"${this.schemaName}"."${this.tableName}"`;
   }
 
+  get computedCollectionTableName() {
```

Comment:
This will return a string regardless of whether a collection table name was defined or not. This should not be an issue, because there is no reason to get this property unless one was passed, but we could change to return undefined.